recipes/tb3py/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "tb3py" %}
+{% set version = "2021.7.11" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tb3py-{{ version }}.tar.gz
+  sha256: 6105173883cd6cbf8544413aedf5e5aa4f47bcaa62be9a8ec8c94a951345f9cd
+
+build:
+  skip: true  # [win]
+  number: 0
+#   noarch: python
```

Comment:
If the only reason this is not noarch and is skipping windows is due to the missing dependency, we can merge this as noarch and ignore the CI failure. The noarch packages are only currently built on linux, and the missing dependency will just result in the package not being installable on Windows.

Reply:
Hi @dopplershift 

Thanks for the review and comments.

I don't necessarily want to use `noarch: python`, the usual one is fine for me. As all the tests are already passing right now, could you please review to merge? In future if it causes any issue, I'll modify the tb3py-feedstock.

Reply:
Well noarch Python is our "usual" for pure-Python packages. So can you explain more the reasoning behind your hesitance making this noarch?

Reply:
@dopplershift  Ok, using noarch-python now, please take a look.